src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -116,10 +116,10 @@ static bool             daos_debug_inited;
 static int              num_dfs;
 static struct dfs_mt    dfs_list[MAX_DAOS_MT];
 
-static void
-discover_daos_mount(void);
-static void
-discover_dfuse(void);
+static int
+discover_daos_mount_with_env(void);
+static int
+discover_dfuse_mounts(void);
```

Comment:
As these two are both defined before they are called in this file then you can simply remove these forward declares now.

Reply:
Removed as suggested. Thank you!